lib/sarif/base_sarif.rb

```diff
@@ -134,16 +134,18 @@ def build_runs_object(supported)
 
         next if !parsed_issue
 
-        next if parsed_issue[:suppressed] && @config['include_suppressed'] == false
+        not_required = (@required.nil? || @required == false)
 
-        next if @required == false && @config['include_suppressed'] == false
+        next if not_required && @config.fetch('include_suppressed', true) == false
+
+        next if not_required && @config.fetch('include_active', true) == false
```

Comment:
It would be better to combine the two lines above into something like the following, to avoid duplicated code

```ruby
x = (!@config.fetch('include_suppressed', true)) ||| !(@config.fetch('include_active', true))
next if not_required && x
```